File: Begginers/Python/imposto_de_renda_1051.py
salario = float(input())

if 0 <= salario <= 2000:
   print(f"Isento")
elif 2000 < salario <= 3000:
   valor = (salario - 2000) * 0.08
   print(f"R$ {valor:.2f}")
elif 3000 < salario <= 4500:
   valor = (salario - 3000) * 0.18 + (1000 * 0.08)
   print(f"R$ {valor:.2f}")
elif salario > 4500:
   valor = (salario - 4500) * 0.28 + (1500 * 0.18) + (1000 * 0.08)
   print(f"R$ {valor:.2f}")

File: Begginers/Java/imposto_de_renda_1051.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double salario, valor;

        salario = sc.nextDouble();

        if (0.0 <= salario && salario <= 2000.0) {
            System.out.println("Isento");
        }
        else if (2000.0 < salario && salario <= 3000.0) {
            valor = (salario - 2000.0) * 0.08;
            System.out.printf("R$ %.2f%n", valor);
        }
        else if (3000 < salario && salario <= 4500) {
            valor = (salario - 3000) * 0.18 + (1000 * 0.08);
            System.out.printf("R$ %.2f%n", valor);
        }
        else if (salario > 4500) {
            valor = (salario - 4500) * 0.28 + (1500 * 0.18) + (1000 * 0.08);
            System.out.printf("R$ %.2f%n", valor);
        }

        sc.close();

    }
}
